documentation/src/main/asciidoc/user_guide/persistence_jdbc.adoc

```diff
@@ -58,7 +58,7 @@ For detailed description of all the parameters used refer to the link:{javadocro
 [source,xml]
 ----
 <persistence>
-   <string-keyed-jdbc-store xmlns="urn:infinispan:config:store:jdbc:9.2" shared="true" fetch-state="false" read-only="false" purge="false">
+   <string-keyed-jdbc-store fetch-state="false" read-only="false" purge="false" key-to-string-mapper="org.infinispan.persistence.keymappers.DefaultTwoWayKey2StringMapper" dialect="H2">
```

Comment:
@ryanemerson looks like `key-to-string-mapper="org.infinispan.persistence.keymappers.DefaultTwoWayKey2StringMapper"` is the default so maybe we should remove it from the example?

Reply:
Yep. We can also remove `fetch-state`, `read-only` and `purge` as those attributes are also using the default values.

Reply:
@ryanemerson thanks. I removed those defaults from the declarative examples + ConfigurationBuilder. I also removed the footnote specific to derby. we dropped that in 10 and should probably do for 9.4 as well. ta.